src/CLICommand/parameters_helper.test.ts

```diff
@@ -29,8 +29,7 @@ import {
 	MetaDataFileParameter,
 	MetaDataGqlTagsParameter,
 	MetadataJsonParameter,
-	IPFSParameter,
-	DataGqlTagsParameter
```

Comment:
Wonder why this is showing up here. DataGqlTags test was added in an adjacent PR not in PE-1880

Reply:
Yup, that was strange. It seems it was me the one that added that line c703ac2